### DISCUSSION TITLE
How can I translate text in DatePicker

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3022] [Q&A] [answered]
### Question

Hi, How can I translate text in DatePicker

I would like to translate the months, days of the week

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Have a look at this: https://python.plainenglish.io/app-localization-in-flet-5b523e83ca89
